app/views/user/show.erb

```diff
@@ -33,6 +33,12 @@
           <% end %>
         <% end %>
       <% end %>
+      <% if profile.can_contribute? %>
+        <h3>Contribute:</h3>
+        <% profile.contributions.each do |k, v| %>
```

Comment:
This is really difficult to understand. This list of contributions is not a list of things that are similar to each other, and I don't think that we should try to DRY it out.